airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/adapters.py

```diff
@@ -280,6 +283,9 @@ class AvailabilityStrategyFacade(AvailabilityStrategy):
     def __init__(self, abstract_availability_strategy: AbstractAvailabilityStrategy):
         self._abstract_availability_strategy = abstract_availability_strategy
 
+    # The data returned by the partitions should be formatted appropriately, so we explicitly set the transformer to NoTransform
+    transformer: TypeTransformer = TypeTransformer(TransformConfig.NoTransform)
```

Comment:
I'm surprised to see this in `AvailabilityStrategyFacade` as I don't see it in the interface `AvailabilityStrategy`. What am I missing? Where is it used? I also do not see a test that explain how it was failing before